Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM alpine:3.5
+
+MAINTAINER ZappiStore DevOps <devops@zappistore.com>
```

Comment:
I know the PR has already closed, but `MAINTAINER` got deprecated in Docker 1.13.
Something to change next time we're here.